src/test/regress/expected/alter_table.out

```diff
@@ -1081,6 +1081,8 @@ drop table atacc1;
 create table atacc1 (a int primary key);
 alter table atacc1 add constraint atacc1_fkey foreign key (a) references atacc1 (a) not valid;
 alter table atacc1 validate constraint atacc1_fkey, alter a type bigint;
+ERROR:  cannot alter column with primary key or unique constraint
+HINT:  DROP the constraint first, and recreate it after the ALTER
```

Comment:
This seems related to the feature not support. Looks OK. Do we have documented this?

Reply:
yes it was documented as ["Note that you cannot alter column data types that are being used as distribution or partitioning keys. "](https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-ref_guide-sql_commands-ALTER_TABLE.html)